cfgov/jinja2/v1/careers/_single.html

```diff
@@ -56,6 +56,7 @@
                 'heading': 'Share this job',
                 'hide_heading': false,
                 'show_linkedin': true,
+                'show_twitter': false,
```

Comment:
@anselmbradford hey ans, now that career pages are rendering on this branch. What twitter options do we want to add?